src/Quartz.Server/Program.cs

```diff
@@ -14,14 +15,16 @@ public static class Program
     /// <summary>
     /// Main.
     /// </summary>
-    public static void Main()
+    public static async Task Main()
```

Comment:
Maybe also could be part of a separate cleanup PR as quite safe and outside of core domain